phpunit.xml.dist

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.0/phpunit.xsd"
```

Comment:
The version number is already managed via the entry in the Composer configuration file, so it is not necessary to add it here.
And the XML schema definition is included in the package itself: https://github.com/sebastianbergmann/phpunit/blob/844d06625f2e8bd40f73e75e220ab1601a0478f8/phpunit.xsd#L2

Reply:
:+1: I upgraded the XML with `--migrate-configuration` and it generated this, I'll rollback the change